runtime/vm/jvminit.c

```diff
@@ -1822,6 +1822,48 @@ processMemoryInterleaveOptions(J9JavaVM * vm)
 	j9port_control(J9PORT_CTLDATA_VMEM_NUMA_INTERLEAVE_MEM, enabled? 1 : 0);
 }
 
+#if JAVA_SPEC_VERSION >= 19
+static VMINLINE IDATA
+processContinuationCacheOptions(J9JavaVM * vm)
+{
+	IDATA rc = 0;
+	IDATA argIndex = FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, VMOPT_XXCONTINUATIONCACHE, NULL);
+	if (-1 != argIndex) {
+		char *cursor = NULL;
+		U_32 cacheSize = 0;
+		GET_OPTION_OPTION(argIndex, ':', ':', &cursor);
+		if (strstr(cursor, "t1=") == cursor) {
```

Comment:
Can we use existing OMR utility helpers such as `try_scan`/`omr_scan_u32`?
In addition, suggest a comment describing the options parsed.